.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   - EVM_EMACS=emacs-git-snapshot-travis
 script:
   - emacs --version
-  - cask exec ert-runner
+  - cask exec ert-runner -L . -L test
```

Comment:
what is the purpose of this?

Reply:
Now `kotlin-mode` requires `kotlin-mode-lexer`, so we must set `load-path` appropriately.  Strictly speaking, `-L test` is not necessary, but it would be useful when the test is spitted into multiple files.